appium-cookies/test/cookies-specs.js

```diff
@@ -0,0 +1,121 @@
+// transpile:mocha
+
+import cookie from '../lib/cookies.js';
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import 'mochawait';
+import 'should';
+
+chai.should();
+chai.use(chaiAsPromised);
+chai.expect();
+let expect = chai.expect;
+
+describe(`trial-test`, () => {
```

Comment:
The descriptions in `describe` and `it` blocks also need some thought. Let's meet offline and go through these.